src/components/apptsSubmitter/ApptsSubmitter.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { ProviderPicker } from '../providerPicker/ProviderPicker';
+import { DatePicker } from '../datePicker/DatePicker';
+import { TimePicker } from '../timePicker/TimePicker';
+
+export const ApptsSubmitter = () => {
+  return (
+    <div style={{border: "1px solid black"}}>
+      <h3>Provider Appontment Submitter Component</h3>
+      <h4>Appointment Range Start</h4>
```

Comment:
The `h4`, `DatePicker`, and `TimePicker` can be moved to a separate component then use that component here.

Reply:
I can incorporate this once I pick a datepicker component. Thanks!